openchrom/plugins/net.openchrom.chromatogram.xxd.report.supplier.pdf.ui/src/net/openchrom/chromatogram/xxd/report/supplier/pdf/ui/swt/ChartScreenshotRunnable.java

```diff
@@ -115,6 +118,24 @@ public void run() {
 			peaks = addPeaks(baseChart, lineSeriesDataList);
 			scans = addScans(baseChart, lineSeriesDataList);
```

Comment:
```suggestion
```
The line series and custom paint listeners are in fact not required.

Reply:
The ILineSeriesData for peaks and scans are required to plot the inverted triangle respectively the dot to mark the peaks / scans. But the PeakLabelMarker and ScanLabelMarker are not needed.